milestone1Project/milestone1.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "milestone1.css">
+    <title>Main Page</title>
+</head>
+<main>
+    <header class = "mainHead">The Grand Recipe Book</header>
+
+
+    <nav class = "subHeaders">
+
+        <a href="cookies.html"><button>Mom's Chocolate Chip Cookies</button></a>
+        <a href ="salad.html"><button>Classic Greek Salad</button></a>
+        <a href = "chicken.html"><button>Elite Chicken and Rice</button>
+        </a>
+    </nav>
```

Comment:
Navbar should be for navigation between your home page with your recipe cards and your about page

Reply:
Missing use of flexbox in navbar